src/plugins/media-slideshow/media-slideshow-editor.js

```diff
@@ -277,6 +294,22 @@ function MediaSlideshowEditor({ content, onContentChanged }) {
                   </div>
                 </FormItem>
                 )}
+                <FormItem
+                  label={
+                    <Fragment>
+                      <Tooltip title={<Markdown>{t('imageFitInfoMarkdown')}</Markdown>}>
+                        <InfoCircleOutlined className="u-info-icon" />
+                      </Tooltip>
+                      <span>{t('imageFit')}</span>
+                    </Fragment>
+                  }
+                  {...formItemLayout}
+                  >
+                  <RadioGroup value={chapters[selectedChapterIndex].image.fit} onChange={handleChapterImageFitChange}>
+                    <RadioButton value="cover">{t('imageFit_cover')}</RadioButton>
+                    <RadioButton value="contain">{t('imageFit_contain')}</RadioButton>
```

Comment:
We should use the `IMAGE_FIT` const here for the values.

Reply:
Yep, fixed (and also the `IMAGE_SOURCE_TYPE` a few lines above)